### ISSUE TITLE
Ajoutez vos suggestions ici !

### ISSUE THREAD

**Issue by Naereen** [Naereen/Bibliographie-agregation-maths-option-info#4]
Si quelqu'un a des suggestions, n'hésitez pas à discuter ici dans les commentaires !

- [ ] Contacter mes anciens camarades 2013-2014 ?
- [ ] Contacter mes copains de l'ENS de 2014-2014 et 2015-2016 ?
- [ ] Contacter mes anciens élèves 2016-2017 ?
- [ ] Contacter mes anciens élèves 2017-2018 ?
- [ ] Contacter mes anciens élèves 2018-2019 ?
- [ ] Contacter mes anciens élèves 2019-2020 ?
- [ ] Contacter des collaborateurs récents ?
- [x] Et des collaborateurs moins récents !
- [x] Et des profs ? Non pas trop.

**Comment by Naereen** [comment]
> Peut-être peux-tu rajouter un livre en info sur les bases de données ? (Le Abiteboul-Hull-Vianu me semble bien par défaut.) J-GL

**Comment by Naereen** [comment]
> Pour le BBC en algo, je l'ai expérimenté cette année en L3, et au moins pour l'algorithmique des graphes, il est parfois assez bizarrement déficient en matière de preuves: du genre, qui commence très rigoureusement puis continue dans l'approximatif, pas du tout à recommander à l'agrég (en résumé: il y a en gros deux extrêmes en preuves d'algo, les preuves algorithmiques usuelles à coup de "on voit bien qu'on ne peut passer qu'une seule fois par sommet en ligne 7", et les preuves par invariants; ni l'une ni l'autre ne sont à recommander à mon avis, pour des raisons opposées de manque de rigueur et de manque de pédagogie, mais le BBC passe parfois de l'un à l'autre en milieu de preuve...)

**Comment by Naereen** [comment]
@LaurentClaessens pour t'informer de ce petit projet, si tu veux jeter un œil.

@johnlepoulpe si tu veux jeter un œil aussi, et en parler à tes camarades.

**Comment by LaurentClaessens** [comment]
Je suis surement de très mauvais conseil pour les livres : je n'en ai aucun. Par contre, je confirme qu'arriver à l'agreg sans livre, c'est du suicide : j'ai fait 1/20 en algèbre et 2/20 en analyse ... mais comme je suis nul en math par ailleurs c'est peut-être simplement la note qui me correspond le mieux :)

Par contre, je vois que "131 développements pour l'oral" ne se gène pas le moins du monde avec les listes de développements. Je vais ressortir [un vieux fichier à moi](https://github.com/LaurentClaessens/mazhe/blob/master/tex/frido/98_liste_developpements.tex) de derrière les fagots.
Le règlement sur les livres est [ici à la page 75](https://agreg.org/data/uploads/rapports/rapport2020.pdf). Il n'y a pas de 
problèmes aux listes de développement.

Attention toutefois à la phrase suivante:
«
 Seuls sont autorisés les ouvrages avec un numéro ISBN et jouissant d’un minimum de diffusion commerciale
»
Elle est écrite au présent. Une lecture stricte montre que les livres qui ne sont plus en vente sont interdits. Dans l'esprit, la suite confirme l'interdiction des livres non trouvables dans le commerce:
«
 Cette restriction est motivée par le principe d’égalité des candidats : les ressources documentaires autorisées doivent être facilement accessibles à tout candidat au concours.
»
Je vois peut-être midi devant ma porte, mais pour moi être présent dans une bibliothèque universitaire n'implique pas "facilement accessible à tout candidat".

**Comment by Naereen** [comment]
Merci de ta réaction très rapide.

Je suis d'accord quant à la difficulté de faire les oraux d'agrég *tels qu'ils existaient avant* sans livre. Les modalités pourraient changer cette année.

Merci du rappel de ces règles dont je suis bien conscient ! Étant un fan absolu du Frido depuis longtemps, l'ayant découvert en 2013 durant ma propre préparation, j'ai suivi les évolutions de ces règles, et tes discussions sur le site du Frido.

 "131 développements pour l'oral" a été écrit spécifiquement pour proposer des listes de développements, et il serait vraiment scandaleux que le jury l'interdise, je n'y crois pas une seconde vu son succès et la qualité du livre.
À noter que ce n'est pas le seul livre très récent qui fasse ça, les auteurs du site <https://agreg-maths.fr/> ont aussi sorti leur livre l'an dernier : https://agreg-maths.fr/livre (et il est aussi très très bien, sauf qu'il ne contient pas d'informatique).

En ce qui concerne le cas du Frido, il serait aussi scandaleux que le jury refuse : c'est un livre diffusé depuis des années, disponible pour un prix très faible en comparaison du volume et de la quantité/qualité du livre.
Si les [conditions annoncées sur le site du jury hier](https://agreg.org/index.php?id=informations-pratiques) restent les mêmes, je vais faire une grosse pub pour ce livre (si tu es d'accord)...

**Comment by LaurentClaessens** [comment]
J'ai été étonné par le "171 ..." parce que j'avais entendu par bruit de couloirs (un commentaire sur linuxfr pour être précis) que c'était le genre de livres qui étaient visés du temps où une liste de livres interdits existait.

Mais comme ce n'est manifestement plus le cas, je ne vais pas me gêner non plus :)

<disclaimer : je en suis pas du tout objectif sur ces questions>
En terme d'équité, le fait que les livres qui ne sont plus édités soient interdits ne me semblerait pas tout à fait débile, et c'est d'ailleurs le cas tant dans l'esprit (affiché) que dans la lettre du règlement.
Cela dit, j'aurais bien du mal à rédiger un règlement qui assure l'équité autrement que "tout est permis pourvu que ce soit sous CC-by-nd ou moins".
</disclaimer>

N'hésite pas à faire la pub. Vu que beaucoup de bibliothèques sont fermées, y'en a qui vont avoir du mal à remplir leurs valises.

**Comment by LaurentClaessens** [comment]
Oh oh. Je viens de me rendre compte d'une chose en relisant le règlement : la règle des 6 mois a disparu.
Avant, les ISBN devaient avoir été attribués au moins 6 mois avant les oraux.
C'est pour cela que j'avais pris l'habitude de publier en décembre 2000+n pour les oraux de juin/juillet 2000+n+1.

**Comment by Naereen** [comment]
Je suis d'accord avec toi, depuis très longtemps, sur ces rêves de règles sur le CC-by-nd etc.
Et chouette nouvelle pour ces règles sur les ISBN !

**Comment by Naereen** [comment]
Tiens un de mes collègues nous a partagé cette page <https://interne.agreg.org/index.php?id=oraux> qui liste environ 35 références qui consisteront la bibliothèque numérique de l'agrégation **interne** de mathématiques.

@LaurentClaessens tu seras heureux de voir "Le Frido" dans la liste ! :clap: :clap: !

C'est d'ailleurs une des trois seules références à ne pas avoir de maison d'édition, avec les deux autres :

- https://hal.archives-ouvertes.fr/hal-01374158, « Probabilités - Préparation à l'agrégation interne, par Djalil Chafaï et Pierre-André Zitt »
- https://cel.archives-ouvertes.fr/cel-00519301/ , « Cours d'analyse par Jacques Harthong »

**Comment by LaurentClaessens** [comment]
Ça c'est une excellente nouvelle.
Le seul inconvénient est qu'ils ne précisent pas la version qui sera disponible; j'ai téléchargé la clef USB, et les documents n'y sont pas. J'imagine que c'est un problème de droits d'auteurs.
En ce qui concerne celui de Chafai, la version qui était disponible en 2015 sur le site de l'agreg et celle qui était effectivement sur les ordinateurs n'était pas la même... Celle dispo in situ était plus récente et plus complète; donc y'avait pas à se plaindre.
Mais en tout cas avec les trucs libres, les problèmes de versions sont à prendre en compte.

Pour le Frido, la version la plus logique serait [celle-ci](http://laurent.claessens-donadello.eu/pdf/Frido2020/lefrido2020.pdf).

**Comment by Naereen** [comment]
Oui, c'est sûr tu as raison. Mais bon même avoir une version de 2019 c'est bien mieux que ne rien avoir !

**Comment by LaurentClaessens** [comment]
Je vais poser la question, tout en suggérant la version de septembre 2020.
